qa/L0_python_api/test_kserve.py

```diff
@@ -29,18 +29,10 @@
 
 import numpy as np
 import pytest
+import testing_utils as utils
```

Comment:
small nit: do we need to rename it or just use "testing_utils", or "test_utils"

Reply:
Initially was `import testing_utils` but changed to `import testing_utils as utils` per @tanmayv25 suggestion. I don't have a strong preference either way.